common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerFactory.scala

```diff
@@ -111,6 +111,20 @@ object ContainerFactory {
   /** include the instance name, if specified and strip invalid chars before attempting to use them in the container name */
   def containerNamePrefix(instanceId: InvokerInstanceId): String =
     s"wsk${instanceId.uniqueName.getOrElse("")}${instanceId.toInt}".filter(isAllowed)
+
+  def resolveRegisterConfig(userProvidedImage: Boolean,
+                            runtimesRegistryConfig: RuntimesRegistryConfig,
+                            userImagesRegistryConfig: RuntimesRegistryConfig): RuntimesRegistryConfig = {
```

Comment:
Wouldn't it make sense to name this function `resolveRegistryConfig()`?

Reply:
yes that makes sense. In fact that was a typo with `resolveRegisterConfig ` ...
thanks for pointing out.